src/main/java/uk/gov/companieshouse/api/strikeoffobjections/email/KafkaRestClient.java

```diff
@@ -0,0 +1,24 @@
+package uk.gov.companieshouse.api.strikeoffobjections.email;
+
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpHeaders;
+import org.springframework.http.HttpMethod;
+import org.springframework.http.ResponseEntity;
+import org.springframework.web.client.RestTemplate;
+
+public class KafkaRestClient {
+
+    private final RestTemplate restTemplate;
+
+    public KafkaRestClient(RestTemplate restTemplate) {
+        this.restTemplate = restTemplate;
+    }
+
+    public byte[] getSchema(String schemaRegistryUrl, String emailSchemaUri) {
+        String schemaUrl = String.format("%s%s", schemaRegistryUrl, emailSchemaUri);
+        HttpHeaders headers = new HttpHeaders();
+        HttpEntity<String> entity = new HttpEntity<>(headers);
+        ResponseEntity<byte[]> response = restTemplate.exchange(schemaUrl, HttpMethod.GET, entity, byte[].class);
```

Comment:
does it work if set the exchange type to String.class? you might be able to get rid of 
String schemaJson = new JSONObject(new String(bytes)).getString("schema");
in KafkaConfiguration then.

Reply:
i mean the new String(bytes) part

Reply:
Really good question -I don't know - hard to test until all subtaks are merged together.

Reply:
we can try at a later date, might be a good little improvement if it works